Modules/+Experiments/+AutoExperiment/@SpecSlowScan/analyze.m

```diff
@@ -191,12 +196,37 @@
         'amplitudes',NaN,...
         'widths',NaN,...
         'locations',NaN,...
+        'etas',NaN,...
         'background',NaN,...
         'index',NaN,... % Index into sites
         'redo',false,...
         'ignore',[]);     % indices of experiments in the fit
 end
 
+if preanalyze
+    err = [];
+    try
+        site_index = 1;
+        progbar = waitbar(site_index/n,'','Name','Analyzing all data','CreateCancelBtn','setappdata(gcbf,''canceling'',1)');
+        setappdata(progbar,'canceling',0);
+        update_all()
```

Comment:
what does this line do/why can't it be in the for loop and start at index 1?

Reply:
changeSite(index) calls save_state(), which needs to have an existing fit added to the figure. Calling update_all with site_index = 1 does the fit first so that changeSite has something to save. This is what was already done on line 232 as well.

Reply:
maybe just toss a comment in there?